OpenSim/Simulation/Test/testInitState.cpp

```diff
@@ -50,6 +50,7 @@ int main()
     try {
         LoadOpenSimLibrary("osimActuators");
         testStates("arm26.osim");
+        //TODO: testMemoryUsage fails, is that okay?
```

Comment:
It either passes saying 0% leak or it fails saying nan leak. I'm assuming it's just a divide by 0 error implying there's no leak?